schema/retries.json

```diff
@@ -28,9 +28,18 @@
           "type": "string",
           "description": "Time delay between retry attempts (ISO 8601 duration format)"
         },
-        "multiplier": {
+        "maxDelay": {
+          "type": "string",
+          "description": "Maximum time delay between retry attempts (ISO 8601 duration format)"
+        },
+        "increment": {
           "type": "string",
-          "description": "Multiplier value by which interval increases during each attempt (ISO 8601 time format)"
+          "description": "Static value by which the delay increases during each attempt (ISO 8601 time format)"
+        },
+        "multiplier": {
+          "type": ["number","string"],
+          "minimum": 0,
+          "description": "Float value, if specified the delay between retries is multiplied by this value."
```

Comment:
I would also define the decimal point precision if you require this to be a float. 
You can use "multipleOf" param when its number type.  Prob enough 1 decimal points in which case add
"multipleOf": 0.1
to multiplier